l10n_it_withholding_tax_reason/__init__.py

```diff
@@ -0,0 +1,30 @@
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+# Copyright 2021 Alex Comba - Agile Business Group
+
+from . import models
+
+from openupgradelib import openupgrade
+
+
+def rename_old_italian_module(cr):
+
+    if not openupgrade.is_module_installed(cr, "l10n_it_withholding_tax_causali"):
+        return
+
+    openupgrade.rename_xmlids(
+        cr,
+        [
+            (
+                "l10n_it_withholding_tax_causali.view_withholding_tax_form_causale",
+                "l10n_it_withholding_tax_causali.view_withholding_tax_form_reason",
+            ),
+        ],
+    )
+
+    openupgrade.update_module_names(
+        cr,
+        [
+            ("l10n_it_withholding_tax_causali", "l10n_it_withholding_tax_reason"),
+        ],
+        merge_modules=True,
```

Comment:
Per evitare l'hook forse questi metodi potrebbero essere aggiunti al blocco nei comandi da shell come fatto [qui](https://github.com/OCA/l10n-italy/pull/2220#issuecomment-828439236) da @tafaRU.

In questo modo non servirebbe `merge_modules=True` (che in OpenUpgrade viene utilizzato nei casi in cui uno o più moduli siano confluiti in un altro nel passaggio di versione) e sarebbe sufficiente aggiornare il modulo. (punto 5. di INSTALL.rst).
Cosa ne pensi @tafaRU ?

Reply:
@primes2h la mia idea iniziale, quando ho iniziato a lavorare a https://github.com/TheMule71/l10n-italy/pull/13, era quella di evitare del tutto le operazioni manuali relative la rename del modulo poi mi sono imbattuto in https://github.com/TheMule71/l10n-italy/pull/13#issuecomment-832583437 e così mi sono trovato costretto a dover spostare quella parte tra le operazioni manuali da eseguire.

Ok, si potrebbe spostare tutto nel blocco dei comandi shell però ciò comporterebbe un ulteriore test. Considerato che il vantaggio che se ne otterrebbe è nullo io sarei per lasciarlo così.

Se invece pensi che sia meglio fare diversamente proponi pure la modifica :smiley:

Reply:
> Ok, si potrebbe spostare tutto nel blocco dei comandi shell però ciò comporterebbe un ulteriore test. Considerato che il vantaggio che se ne otterrebbe è nullo io sarei per lasciarlo così.

:+1:  
Lasciamolo pure così